galley/pkg/config/source/kube/interfaces.go

```diff
@@ -18,7 +18,9 @@ import (
 	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp" // import GKE cluster authentication plugin
+
```

Comment:
This is not right

Reply:
That is true, looks like it has an extra space. But as long as the tool is consistent may not be a big issue?

Reply:
I will fix it :)